doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -469,6 +479,13 @@ tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
 rm -f freemarker.tgz
```

Comment:
in the above `tar` cmd, `--strip=2` needs to be changed to `--strip-components=2`.

Reply:
Will change.